client/src/components/navbar/mobileNavbar.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect, useContext } from "react";
+import {
+  IconButton,
+  Button,
+  Box,
+  Dialog,
+  DialogActions,
+  Grid,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+import { Link } from "react-router-dom";
+import { UserContext } from "../../context/Context";
+import Slide from "@material-ui/core/Slide";
+
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Slide direction="right" ref={ref} {...props} />;
+});
+
+const useStyles = makeStyles(theme => ({
+  root: { flexgrow: 1 },
+  closeButton: {
+    position: "absolute",
+    right: theme.spacing(1),
+    top: theme.spacing(1),
+    color: theme.palette.grey[500],
+  },
+  gridElement: {
+    maxWidth: "350px",
+    width: "85%",
+    textAlign: "center",
+    height: "80px",
+    margin: "30px auto 30px",
+  },
+  menuButton: {
```

Comment:
You'll probably want to get rid of the underline for these as well

Reply:
Okay. I just personally thought It looked better with them :)

Reply:
Ah okay, feel free to stick with it then since there's no mocks for the mobile view. I wasn't sure if it was intentional or not